components/dashboard/src/workspaces/CreateWorkspacePage.tsx

```diff
@@ -461,6 +467,64 @@ export function CreateWorkspacePage() {
     );
 }
 
+function LocalSSHWarning(props: { className: string }) {
+    const isLocalSSHProxyEnabled = useFeatureFlag("gitpod_desktop_use_local_ssh_proxy");
+    const [isFeedbackFormVisible, setFeedbackFormVisible] = useState<boolean>(false);
+
+    if (!isLocalSSHProxyEnabled) {
+        return null;
+    }
+
+    return (
+        <div className={props.className}>
+            <Alert
+                light
+                className="bg-gray-100 dark:bg-gray-800"
+                type="info"
+                iconSize="w-8 h-8"
+                icon={<Exclamation2 className="h-full w-full" />}
```

Comment:
**question**: Do we want to override the component type? The info alert component variant is already using a ℹ️ icon, right? Added also some copy changes for inspiration.

| BEFORE | AFTER |
|-|-|
| <img width="601" alt="Frame 1625" src="https://github.com/gitpod-io/gitpod/assets/120486/a54d66a1-f8fe-449d-b1e4-d4fa5f82a28f"> | <img width="601" alt="Frame 1626" src="https://github.com/gitpod-io/gitpod/assets/120486/548e4357-f6e2-4b2f-8e8d-ec6dae36e6f6"> |

Reply:
@gtsiolis  I use custom style because `warning` icon can bring more attention, and not orange background color to not panic users.

But yep, I can switch to use info level of Alert, and make icon size default